src/popper/translators/translator_drone.py

```diff
@@ -7,31 +7,69 @@ def __init__(self):
         super().__init__()
 
     def translate(self, wf):
-        box = Box(kind="pipeline", type="docker", name="default")
+        # get workflow type (Docker or Exec)
+        wf_type = self._detect_type(wf)
+
+        box = Box(kind="pipeline", type=wf_type, name="default")
 
         # environment variables available throughout the pipeline
         wf_env = {
             "GIT_COMMIT": "${DRONE_COMMIT_SHA}",
             "GIT_BRANCH": "${DRONE_COMMIT_BRANCH}",
             "GIT_SHA_SHORT": "${DRONE_COMMIT_SHA:0:7}",
             "GIT_REMOTE_ORIGIN_URL": "${DRONE_GIT_HTTP_URL}",
-            "GIT_TAG": "${DRONE_TAG}",
+            "GIT_TAG": "${DRONE_TAG}:-''",  # use the empty string if the variable is undefined
```

Comment:
Drone's exec runner does not have `GIT_TAG` and crashes without the default value